versions.lock

```diff
@@ -89,11 +89,11 @@ com.palantir.dialogue:dialogue-core:3.69.0 (3 constraints: 883d5fe3)
 com.palantir.dialogue:dialogue-futures:3.69.0 (3 constraints: dd33d7c0)
 com.palantir.dialogue:dialogue-serde:3.69.0 (3 constraints: c62e7ca5)
 com.palantir.dialogue:dialogue-target:3.69.0 (7 constraints: a776356a)
-com.palantir.docker.compose:docker-compose-rule-core:1.8.0 (3 constraints: d92f0e11)
+com.palantir.docker.compose:docker-compose-rule-core:1.8.0 (3 constraints: dc2f1711)
```

Comment:
noting to self that we should merge the other PR #6349 (if no one gets to it tomorrow, I'll do it) and then merge here so this isn't in the diff